azure-pipelines.yml

```diff
@@ -351,7 +351,7 @@ stages:
           docker pull libertem/libertem-dev || true
           docker build --cache-from=libertem/libertem-dev -t libertem/libertem-dev --build-arg BUILDKIT_INLINE_CACHE=1 ../../ -f Dockerfile.dev
           # make this succeed even if the access token is not available:
-          (docker login -u sk1p -p $(DOCKER_ACCESS_TOKEN) && docker push libertem/libertem-dev) || true
```

Comment:
FYI this worked fine just yesterday, as evident from the [last `libertem/libertem-dev` push](https://hub.docker.com/r/libertem/libertem-dev)

Reply:
Hm strange, I got this: https://dev.azure.com/LiberTEM/LiberTEM/_build/results?buildId=1394&view=logs&j=ced89aaa-f671-5884-bb56-a0d56c033fe1&t=2127ab87-9cad-521d-7c58-c5a20da19f7f&l=1634

Reply:
Ah, see comment below about only `master` having access to tokens

Reply:
Ah OK! I'll change it back and test properly if we want to run etc